docs/_sources/tutorial.txt

```diff
@@ -0,0 +1,458 @@
+Dynamic REST Tutorial
+*********************
+
+.. contents::
+
+.. _intro:
+
+Introduction
+=====================
+
+This document will take you through the basics (and not-so-basics) of developing Dynamic REST APIs. `Dynamic REST <https://github.com/AltSchool/dynamic-rest>`_ (or “DREST”) is an extension to `Django REST Framework <http://www.django-rest-framework.org/>`_ (“DRF”) that makes it easier to implement uniform APIs with powerful features like field inclusion/exclusion, flexible filtering, pagination, and efficient "sideloading" of related/nested data.
+
+Setup
+-----
+
+Before starting the tutorial, you’ll need to set up Dynamic REST in your dev environment. Refer to the README for details, but it should look something like::
+
+    $ git clone git@github.com:AltSchool/dynamic-rest.git
+    $ cd dynamic_rest
+    $ CMD=migrate make run
+    $ CMD=shell make run
+    >>> from tests.setup import create_fixture
+    >>> create_fixture()
+    $ make server
```

Comment:
We can probably add a make wrapper to do all of this, similar to what is currently happening in the Procfile: `manage.py migrate --settings=tests.settings && manage.py runserver 0.0.0.0:$PORT --settings=tests.settings`